Sources/Common/UserInfo.swift

```diff
@@ -0,0 +1,166 @@
+//
+//  UserInfo.swift
+//
+//  Copyright © 2022 DuckDuckGo. All rights reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+import Foundation
+
+
+/**
+ Arbitrary User Info storage with default values. Equatable. Private extension keys do NOT intersect between extensions.
```

Comment:
It would be nice to mention in the docs that it’s used by Navigation code to pass arbitrary data in navigation actions (especially that UserInfo is defined in Common and Navigation is in BSK).